src/main/java/org/broadinstitute/hellbender/tools/spark/sv/StructuralVariationDiscoveryPipelineSpark.java

```diff
@@ -110,14 +111,14 @@
             = new DiscoverVariantsFromContigsAlignmentsSparkArgumentCollection();
     @Argument(doc = "sam file for aligned contigs", fullName = "contig-sam-file")
     private String outputAssemblyAlignments;
-    @Argument(doc = "filename for output vcf", shortName = StandardArgumentDefinitions.OUTPUT_SHORT_NAME,
+    @Argument(doc = "prefix for output vcf; sample name will be appended after the provided argument",
```

Comment:
Is this assuming that if you want files to appear in a directory then the argument should end with a trailing slash? I'd like to avoid that if possible. Can we handle this with eg the Path API? 

I.e. Let's make this actually call for a directory. Then we should validate that the directory actually exists and create it if it's not there. Finally we can create the files in that directory using an API call, like `Paths.resolve(outputDirectory, filename)` for each filename we want.

Reply:
is the commit 2fd080dcd17b4d19f9efbed8028bb44f631db793 similar to what you have in mind?